### ISSUE TITLE
Add Nodejs Tutorials

### ISSUE THREAD

**Issue by mani1soni** [mani1soni/edu_adda#31]

**Comment by Leoperon** [comment]
are you planning to build a project for the tutorial

**Comment by jrajni** [comment]
can i take this issue?

**Comment by Leoperon** [comment]
We can work together. As I am already working on it 🙂

**Comment by fadhilnurmahardi** [comment]
i already add tutorial in nodejs, please merge my PR #35 @mani1soni

**Comment by mani1soni** [comment]
> can i take this issue?

yeah

**Comment by mani1soni** [comment]
> i already add tutorial in nodejs, please merge my PR #35 @mani1soni

okay

**Comment by mani1soni** [comment]
> are you planning to build a project for the tutorial

yeah, but I don't have much knowledge of nodejs, so if you have, you can share your knowledge with others.

**Comment by NazimFilzer** [comment]
What has to be done here?
